resources/user-agents/browsers/whale/whale.json

```diff
@@ -79,6 +79,33 @@
           ]
         }
       ]
-    }
+    },
+    {
+        "userAgent": "Whale Browser for Android",
+        "browser": "whale browser",
+        "platform": "Android",
+        "engine": "Blink",
+        "device": "general Mobile Device (Touch)",
```

Comment:
Please use `general Mobile Phone (Touch)` as default and remove this from the child.

Reply:
Thank you for your review. I have corrected that you pointed.